molecule/default/requirements.yml

```diff
@@ -7,5 +7,7 @@ collections:
     version: ">=3.10.2"
 
 roles:
+  - name: skeleton
+    src: https://github.com/cisagov/skeleton-ansible-role
```

Comment:
What does this mean for testing on branches?

Reply:
You raise a very good point that I will have to think about.

Reply:
See #220 as an interim solution that I will pursue if I can't find a solution along the lines of this PR.